de.klayout.KLayout.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The metainfo should be contributed upstream if possible

Reply:
Okay, I submitted it here: https://github.com/KLayout/klayout/pull/562